net/curl/Makefile

```diff
@@ -8,14 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=curl
-PKG_VERSION:=7.83.1
+PKG_VERSION:=7.84.0
 PKG_RELEASE:=$(AUTORELEASE)
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=https://dl.uxnr.de/mirror/curl/ \
-	https://curl.askapache.com/download/ \
-	https://curl.se/download/
-PKG_HASH:=2cb9c2356e7263a1272fd1435ef7cdebf2cd21400ec287b068396deb705c22c4
+PKG_SOURCE_URL:=https://curl.se/download/
```

Comment:
Default to GitHub and add curl.se as fallback
https://github.com/curl/curl/releases/tag/curl-7_84_0

Reply:
Great idea, is there a way to convert 7.84.0 to 7_84_0 in the Makefile as to not to hardcode the specific version in the URL?